tsnet/tsnet.go

```diff
@@ -180,11 +182,19 @@ func (s *Server) start() error {
 
 	// TODO(bradfitz): start logtail? don't use filch, perhaps?
 	// only upload plumbed Logf?
```

Comment:
What @dsnet said (assumptions about globals and only being run once early in the life of the program) and this comment here about "only upload plumbed logf" is why this bug is annoying and I skipped it all earlier :)

Reply:
@dsnet and I paired for an hour this morning, so I'll be pushing up more changes later.  I think things are in much better shape now.